dsalgo/linked_list.py

```diff
@@ -40,7 +40,7 @@ def to_list(self):
 
 		node = self.head
 		while node:
-			out_list.append(node.value)
+			out_list.append([node.value,hex(id(node))])
```

Comment:
add space after comma (can use flake 8)

Reply:
well that's some unneccessary change and i use pylinter